pkg/types/block_state.go

```diff
@@ -26,11 +27,63 @@ type StateBlock struct {
 	Timestamp      int64     `msg:"timestamp" json:"timestamp"`
 	Extra          Hash      `msg:"extra,extension" json:"extra,omitempty"`
 	Representative Address   `msg:"representative,extension" json:"representative"`
-	Work           Work      `msg:"work,extension" json:"work"`
-	Signature      Signature `msg:"signature,extension" json:"signature"`
+
+	PrivateFrom    string   `msg:"priFrom,omitempty" json:"privateFrom,omitempty"`
+	PrivateFor     []string `msg:"priFor,omitempty" json:"privateFor,omitempty"`
+	PrivateGroupID string   `msg:"priGid,omitempty" json:"privateGroupID,omitempty"`
+
+	Work      Work      `msg:"work,extension" json:"work"`
+	Signature Signature `msg:"signature,extension" json:"signature"`
+}
+
+func (b *StateBlock) BuildHashData() []byte {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
ST1016: methods on the same type should have the same receiver name (seen 36x "b", 5x "z") (stylecheck)